axelrod/strategies/neg.py

```diff
@@ -0,0 +1,35 @@
+import random
+from axelrod import Actions, Player, random_choice, flip_action, init_args
+from axelrod.strategy_transformers import TrackHistoryTransformer
+
+C, D = Actions.C, Actions.D
+
+class Neg(Player):
+    """
+    A player starts by cooperating or defecting randomly if it's their first move,
+     then simply doing the opposite of the opponents last move thereafter.
+
+    Names:
+
+    Neg - [No official reference found in docs, only found in 'Desired New Strategies' list: https://github.com/Axelrod-Python/Axelrod/issues/379]
```

Comment:
Please list this as the reference: http://www.prisoners-dilemma.com/competition.html

Note that the full strategy name is `Negation`, and we should use that as the class name.